cleansio/cleansio.py

```diff
@@ -12,16 +14,40 @@ def valid_input():
 def cleanup():
     """ Removes temporary files """
     if 'CLEANSIO_TEMP_FILE' in os.environ:
-        os.remove(os.environ.get('CLEANSIO_TEMP_FILE'))
+        print("Removing temp file")
+
+        try:
+            os.remove(os.environ.get('CLEANSIO_TEMP_FILE'))
+        
+        except FileNotFoundError as fnfe:
+            print("File {0} doesn't exist".format(os.environ.get('CLEANSIO_TEMP_FILE')))
+
+        print("Removed temp file {0}".format(os.environ.get('CLEANSIO_TEMP_FILE')))
+
     if 'CLEANSIO_SLICES_LIST' in os.environ:
         slices_list_env_var = os.environ['CLEANSIO_SLICES_LIST']
         slices_list = slices_list_env_var[2:-2].split("', '")
         for slice_file in slices_list:
-            os.remove(slice_file)
+            print("Removing slice {0}".format(slice_file))
+
+            try:
+                os.remove(slice_file)
+        
+            except FileNotFoundError as fnfe:
+                print("Couldn't find slice file {0}".format(slice_file))
```

Comment:
Since fstrings are already being used in the other file, is there any reason why you wouldn't use fstrings for these too?

Reply:
We are using Python < 3.6, so we decided to change back to .format() statements.